### PR TITLE
Create named inline script

### PR DESCRIPTION
**Pull request by YakDriver**

### CODE REVIEW

**Comment by lorengordon** [comment]
go codebuild go

**Comment by YakDriver** [comment]
go codebuild go

**Comment by YakDriver** [comment]
On Linux there are two inline scripts: a mini one and the main one that the mini runs. I didn't change the name of the mini since it's mini but I will.

**Comment by YakDriver** [comment]
go codebuild go

File: main.tf [hunk 1]
```diff
@@ -194,20 +194,20 @@ resource "aws_instance" "lx" {
...
+      "chmod +x ~/inline-${local.ami_underlying[element(local.lx_requests, count.index)]}.sh",
+      "~/inline-${local.ami_underlying[element(local.lx_requests, count.index)]}.sh",
     ]
 
     connection {
       host = coalesce(self.public_ip, self.private_ip)
       type = "ssh"
       # this is where terraform puts the above inline script
-      script_path = "~/inline.sh"
+      script_path = "~/inline-mini-${local.ami_underlying[element(local.lx_requests, count.index)]}.sh"
```
Comment (lorengordon) [inline main.tf:210]: bam! can we do that also for the winrm connection on windows?

Comment (YakDriver) [reply main.tf:217]: This should work...

**Comment by YakDriver** [comment]
go codebuild go

**Pull request merged**